.github/workflows/test.yml

```
@@ -15,13 +15,19 @@ jobs:
     name: Test (OTP ${{ matrix.otp }} / Elixir ${{ matrix.elixir }})
     strategy:
       matrix:
-        elixir: ['1.15', '1.14', '1.13', '1.12']
-        otp: ['26', '25', '24', '23']
+        elixir: ["1.16", "1.15", "1.14", "1.13", "1.12"]
+        otp: ["26", "25", "24", "23", "22"]
```

```suggestion
        otp: ["26", "25", "24", "23"]
```
Maybe there is no need to add OTP version 22 based on the previous workflow version.

---

I've added all supported elixir versions on all supported otp releases: https://hexdocs.pm/elixir/1.16.0/compatibility-and-deprecations.html